widgets/contact/fields/radio.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class SiteOrigin_Widget_ContactForm_Field_Radio extends SiteOrigin_Widget_ContactForm_Field_Base {
+
+	public function render_field( $options ) {
+		if ( ! empty( $options['field']['options'] ) ): ?>
+		<ul>
+			<?php foreach ( $options['field']['options'] as $i => $option ): ?>
+			<li>
+				<label>
```

Comment:
Missed this, there should be a closing `<label>` tag too. Also, double check the indenting for this section. The `foreach` and the `if` should indent the next lines. So make sure that the opening and closing `<ul>` and `<li>` are at the same depth.